### PR TITLE
Add email validation 

### PR DESCRIPTION
**Pull request by Aquinasmoriasi**
- Added email validation
- Add error message
- Added timeout to error message
- Add styling to error message

### CODE REVIEW

**Comment by Ismailco** [comment]
@Aquinasmoriasi, @29td kindly fix linters

File: script.js [hunk 1]
```diff
@@ -47,3 +47,226 @@ navLink3.addEventListener('click', () => {
...
+// const email = document.getElementById('email');
+// const error = document.querySelector('.error');
+// const isLowerCase = (str) => str === str.toLowerCase();
+// document.getElementById('submit').addEventListener('submit', (event) => {
+//   if (!isLowerCase(email)) {
+//     event.preventDefault();
+//     error.innerHTML = 'Please enter a lowercase email';
+//   }
+// });
+
```
Comment (tresorsawasawa) [inline script.js]: - [ ] Good job team, your project works as expected. But make sure to remove the commented code lines lines to make your codebase cleaner

File: script.js [hunk 2]
```diff
@@ -47,3 +47,226 @@ navLink3.addEventListener('click', () => {
...
+    error.innerHTML = 'Please write a lowercase email';
+    error.style.color = 'red';
+    setTimeout(() => {
+      error.innerHTML = '';
+    }, 5000);
+  }
+});
+
+
+
```
Comment (tresorsawasawa) [inline script.js]: - [ ] I think it would be better if you remove the unnecessary empty lines in your code to make it cleaner

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Team
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishimwezachee — APPROVED** [review]
Hi @Aquinasmoriasi  and @tresorsawasawa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
All linters are passing ✔️
Good commit history ✔️
Your design looks good 💯 ✔️
Good PR description ✔️
Follow GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**